diuid-docker-proxy/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"strconv"
+	"syscall"
+)
+
+const (
+	realProxy       = "docker-proxy"
+	diuidParentHost = "10.0.2.2"
+)
+
+// drop-in replacement for docker-proxy.
+// forked from https://github.com/rootless-containers/rootlesskit/blob/v0.6.0/cmd/rootlesskit-docker-proxy/main.go
```

Comment:
note: the original file was licensed under Apache2, but as I'm the only one author of the original file, I'm ok to put this file to diuid repo under GPL3